src/frontend/src/utils/users.ts

```diff
@@ -9,3 +9,13 @@ import { fullNamePipe } from './pipes';
 export const userToAutocompleteOption = (user: User): { label: string; id: number } => {
   return { label: `${fullNamePipe(user)} (${user.email})`, id: user.userId };
 };
+
+/**
+ * Determines whether a user is authorized to view the Admin Tools page.
+ *
+ * @param user the user to check
+ * @returns whether they can view Admin Tools
+ */
+export const canAccessAdminTools = (user?: AuthenticatedUser): boolean => {
+  return isHead(user?.role) || (isLeadership(user?.role) && user?.isFinance) || false;
```

Comment:
Similar to the comment in AdminToolsPage, I think the ticket said that all leads and heads can see the admin tools page. Heads and leads only need to be in the finance team if they want to modify vendors, but it would be better to confirm with Reid since he created this ticket. @RChandler234 Can you confirm this?

Reply:
That is what the ticket says, but tbh with what I know about the difference between heads vs. leads in the team now, it makes sense for only finance leads to be able to even see the page